content/02.main-text.md

```diff
@@ -432,7 +432,14 @@ Manubot provides an ideal platform for perpetual reviews [@arxiv:1502.01329; @ta
 
 Concepts for the future of scholarly publishing extend beyond collaborative writing [@doi:10.22541/au.148769949.92783646; @doi:10.22541/au.149693987.70506124; @tag:techblog-brown].
 Pandoc Scholar [@doi:10.7717/peerj-cs.112] and Bookdown [@doi:10.1201/9781315204963], which has been used for open writing [@doi:10.12688/f1000research.16516.1], both enhance traditional Markdown to better support publishing.
-The [knitcitations](https://github.com/cboettig/knitcitations) package enables citation by DOI or URL in R Markdown documents.
+[Dokieli](https://dokie.li/) stores articles as HTML and includes a clientside editor to allow reader annotation and interaction [@doi:10.1007/978-3-319-60131-1_33].
```

Comment:
@csarven is this an accurate description of dokieli?

Reply:
Thanks for the mention.

I suppose that you would just likely to briefly acknowledgement of dokieli's existence FWIW (and nothing particularly beyond that from the looks of it), so I think the current tagline "dokieli is a clientside editor for decentralised article publishing, annotations and social interactions." will suffice for your needs.

As for "doi:10.1007/978-3-319-60131-1_33", that's a bit like junk food. Please use https://csarven.ca/dokieli-rww or https://csarven.ca/linked-research-decentralised-web#decentralised-linked-research-application instead.

Reply:
Updated the citation in https://github.com/greenelab/meta-review/pull/255/commits/837900f22fec95c70f4c426d8501e686f2feab24

Note that `manubot cite url:https://csarven.ca/dokieli-rww` returns incomplete CSL JSON metadata for the URL:

```json
[
  {
    "id": "q41cceRt",
    "type": "webpage",
    "title": "Decentralised Authoring, Annotations and Notifications for a Read-Write Web with dokieli",
    "URL": "https://csarven.ca/dokieli-rww",
    "accessed": {
      "date-parts": [
        [
          "2019",
          9,
          6
        ]
      ]
    },
    "note": "This CSL JSON Item was automatically generated by Manubot v0.2.3 using citation-by-identifier.\nstandard_id: url:https://csarven.ca/dokieli-rww"
  }
]
```

Therefore I had to manually create the CSL JSON metadata. Dokieli could set HTML `<meta>` values [in the head](https://www.zotero.org/support/dev/exposing_metadata) such that the metadata would be picked up automatically.

Reply:
Thanks for the update!

Why resort to proprietary solutions?

If you need *data* along the lines of JSON, ie. in JSON-LD, is this sufficient http://rdf.greggkellogg.net/distiller?command=serialize&output_format=jsonld&url=https:%2F%2Fcsarven.ca%2Fdokieli-rww&raw ?

You can transform the HTML+RDFa into other RDF syntaxes if you prefer.